### PR TITLE
chore:Lambda path fix for data colletor lambda

### PR DESCRIPTION
**Pull request by rnithinpaladin**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List
Lamaba path fix 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by mkinchin-paladin — APPROVED** [review]

**Comment by plyubich** [comment]
@rnithinpaladin pleasw name your beach something like fix/devops-123 and add the issues id to the comment as [devops-122]. This way your commit get automatically attached to the Jira ticket.

What story is this related to?

File: installer/resources/lambda_submit/function.py [hunk 1]
```diff
@@ -804,7 +804,7 @@ class LongRunningLambdaFunction(LambdaFunctionResource):
     handler = BATCH_LONG_RUNNING_JOB_FILE_NAME + ".lambda_handler"
     runtime = "python3.8"
     s3_bucket = BucketStorage.get_output_attr('bucket')
-    s3_key = UploadLambdaSubmitJobZipFile.get_output_attr('id') 
+    s3_key = UploadLambdaLongRunningJobZipFile.get_output_attr('id')
```
Comment (plyubich) [inline installer/resources/lambda_submit/function.py:807]: How is this related to the data collectors?

**Review by plyubich — CHANGES_REQUESTED** [review]

File: installer/resources/lambda_submit/function.py (hunk 1 above)
Comment (rnithinpaladin) [reply installer/resources/lambda_submit/function.py:807]: DataColletor lambda is picking the wrong code from s3.

**Review by santhosh-challa — APPROVED** [review]
